content/touchpad.md

```diff
@@ -20,9 +20,15 @@ tableOfContents: true
 
 Press <kbd>Fn</kbd>+<kbd>F1</kbd> to turn your laptop touchpad on/off.
 
+>**Note**: This shortcut is not available on the Pangolin (pang12).
```

Comment:
The Pangolin does have a touchpad disable shortcut, it's just Fn-F6 instead of Fn-F1: https://tech-docs.system76.com/models/pang12/external-overview.html

Reply:
Should be fixed in the latest commit!